setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import ast
+import os.path
+import re
+from codecs import open
+
+from setuptools import setup
+
+ROOT = os.path.realpath(os.path.dirname(__file__))
+init = os.path.join(ROOT, 'data_api', '__init__.py')
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+_name_re = re.compile(r'NAME\s+=\s+(.*)')
+
+with open(init, 'rb') as f:
+    content = f.read().decode('utf-8')
+    VERSION = str(ast.literal_eval(_version_re.search(content).group(1)))
+    NAME = str(ast.literal_eval(_name_re.search(content).group(1)))
+
+
+def get_requirements(filename):
+    with open(ROOT + '/' + filename) as requirements_file:
+        install_requires = requirements_file.read().splitlines()
+    return install_requires
+
+
+setup(
+    name=NAME,
+    version=VERSION,
+    author='UNICEF',
+    author_email='rapidpro@unicef.org',
```

Comment:
is this a valid email?

Reply:
do you have a valid email?

Reply:
I think this is good actually. I believe it's a shared account evan can access.